src/main/java/net/sf/jabref/gui/prrv/PrrvDialogView.java

```diff
@@ -0,0 +1,280 @@
+        titleCB.setText("Show all keys");
+        //attach click-method to all 3 checkboxes
+        keyCB.setOnAction(e -> handleCheckBoxAction(e));
+        titleCB.setOnAction(e -> handleCheckBoxAction(e));
+        vbox.getChildren().addAll(keyCB, titleCB);
+        return vbox;
+    }
+
+    private void handleCheckBoxAction(ActionEvent e) {
+        if(keyCB.isSelected()){
+            DisplayControl dc = new DisplayControl();
+            Iterator<VisualItem> it = vis.items();
+
+
+            System.out.println("Key activated");
+        }
+        else {
+            System.out.println("Key deactivated");
+        }
+        if(titleCB.isSelected()){
+            System.out.println("Title activated");
+        }
+        else {
+            System.out.println("Title deactivated");
+        }
+    }
+
+    private static void showGuideDialogs() {
+
+        boolean makeCiteFields = false;
+        boolean considerYourDocument = false;
+        String mainDocument = "Your document";
+        List<BibEntry> pureEntryList = JabRefGUI.getMainFrame().getCurrentBasePanel().getBibDatabaseContext().getDatabase().getEntries();
+
+        // ----- First dialog with option to create cites field as optional field
+        String title = "Error while loading cites";
+        String content = "All used entry types need an optional field named cites\n"
+                + "Should JabRef insert them automatically?\n"
+                + "Note:\n "
+                + "- You can insert entry types under BibTeX < Customize entry types.\n"
+                + "- BibTex entries which are not cited are not displayed.";
+        Optional<ButtonType> buttonType = FXDialogs.showCustomButtonDialogAndWait(Alert.AlertType.INFORMATION, title, content, ButtonType.OK, ButtonType.CANCEL);
+        if (buttonType.isPresent() && buttonType.get() == ButtonType.OK) {
+            if (!pureEntryList.isEmpty()) {
+                makeCiteFields = true;
+            }
+            primaryStage.close();
+        } else {
+            primaryStage.close();
+        }
+
+        // ----- Second dialog which asks to use your document as root node
+        if(makeCiteFields) {
+        Optional<ButtonType> rootNodeButtonType = FXDialogs.showCustomButtonDialogAndWait(Alert.AlertType.INFORMATION,
+                "Your document as root node", "Should JabRef consider your document as a root node of the visualization?",
+                ButtonType.OK, ButtonType.CANCEL);
+        if (rootNodeButtonType.isPresent() && buttonType.get() == ButtonType.OK) {
+            mainDocument = FXDialogs.showTextInputDialogAndWait("Name" +
+                    " of your document", "Document name","Please write the name of your document");
+            considerYourDocument = true;
+        }
+
+        // ----- Apply cite fields and root node in consideration of made decisions
+
+            // Setup new cites field
+            for (int id = 0; pureEntryList.size() > id; id++) {
```

Comment:
Please use a foreach loop or a stream (if possible) if you only use the counter variable to get the element at its position.